.gitignore

```diff
@@ -13,3 +13,4 @@ junit-report.xml
 .deptrac.cache
 .phpunit.result.cache
 infection.log
+tests/OutputFormatter/data/graphviz.dot
```

Comment:
this file is generated when running tests locally.. this file is not meant to be committed into the repo we ignore it.